libfwupd/fwupd-client.c

```diff
@@ -2473,7 +2476,7 @@ fwupd_client_install_release_download_cb (GObject *source, GAsyncResult *res, gp
 }
 
 static gboolean
-fwupd_client_is_url (const gchar *perhaps_url)
+fwupd_client_is_url_http (const gchar *perhaps_url)
```

Comment:
doesn't this function also handle directories?

Reply:
I don't think so; `fwupd_client_install_release_remote_cb` uses it as `fwupd_remote_get_kind (remote) == FWUPD_REMOTE_KIND_LOCAL && !fwupd_client_is_url_http (fwupd_release_get_uri (data->release)` i.e. a directory would return `FALSE` from `fwupd_client_is_url()`